woocommerce-onpay.php

```diff
@@ -439,6 +439,7 @@ public function order_meta_box($post, array $meta) {
                 $html .= '<th>' . __('IP', 'wc-onpay') . '</th>';
                 $html .= '</thead><tbody>';
                 foreach ($transaction->history as $history) {
+                    $history->dateTime->setTimeZone(wp_timezone());
```

Comment:
Something whitespace related is weird here ?

Reply:
Actually it's the lines underneath that's only indented with 3 spaces